common/src/main/java/com/wynntils/models/character/CharacterModel.java

```diff
@@ -138,6 +149,19 @@ private void scanCharacterInfoPage() {
         query.executeQuery();
     }
 
+    private void scanCratesBombsCosmeticsPage() {
+        ScriptedContainerQuery query = ScriptedContainerQuery.builder("Crates, Bombs & Cosmetics Query")
```

Comment:
The key for the query is actually important, has to be unique. I suggest adding ": Parsing Silverbull Status", so we fulfill that.

Reply:
Or better only "Getting Silverbull Subscription"